packaging/centos-7/build-inside/build-package

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -ex
+# we should always set proper ownership before exiting, otherwise
+# the created packages will have root:root ownership and we'll be unable
+# to delete them from our host.
+trap 'chown -R --reference /build-inside/build-package /out/' EXIT
+
+# the source directory is mounted read-only to prevent issues where the build
+# could alter the source; we should copy it somewhere inside the container
+cd /tmp
+rsync -a /source .
+cd source
+make
+mkdir /tmp/installdir
+make DEST_DIR=/tmp/installdir/usr/local install
+
+# signing block. can be omitted if you don't need to sign your RPMs.
+# gpg --import /tmp/private.key
+# [[ $(gpg --list-secret-keys) =~ uid(.*) ]]
+# KEYNAME="${BASH_REMATCH[1]}"
+# [ -n "${KEYNAME}" ] || { echo "could not find key for signing purpose"; exit 1; }
+# echo -e "%_gpg_name ${KEYNAME}\n%_signature gpg\n" > ${HOME}/.rpmmacros
+# signing block end
+
+cd /out
+# this is a standard fpm command. look at fpm's own help. The only thing that I like to do
+# is installing all the things in a separate dir to prevent listing all the files - but it's
+# just personal taste.
+# don't forget to list all runtime dependencies for your package (i.e. all the shared libraries you link to!)
+# the --rpmsign flag can be omitted if you don't need signing your package.
+# the "setsid" magic is used to prevent rpmsign asking for a password.
+# echo -e "\n" | setsid fpm -t rpm -s dir -n osuosl-dssp --rpm-sign --version ${VERSION} --description "OSUOSL built DSSP" --depends readline -C / /usr/local/dssp
+
+#this would be the no-signature command line
+fpm -t rpm -s dir -n osuosl-dssp --vendor "OSU Open Source Lab" --license "Boost License 1.0" --url "http://www.cmbi.ru.nl/dssp.html" --version ${VERSION} --description "OSUOSL built DSSP" -C /tmp/installdir usr/local
```

Comment:
My editor yells at me for long lines in Python but not in shell, so I hadn't noticed just how long tha tline was.  It's now wrapped within the 80 character margin.